server/.env

```diff
@@ -0,0 +1,6 @@
+DATABASE_URI=mongodb://localhost:27017
```

Comment:
the entire point of ignoring the `.env` file is so that we don't leave secrets out in public right? That's why there's a `.env.example` file instead, which requires the user to fill it out themselves (by reaching out to the team).

We would want this file gone, at least not commited :P

Reply:
Agreed. But, this file doesn't contain any secrets, and it is not a valid .env file.
I was thinking this would be a useful starting point for people, since it took me a little while to figure out what all the variables meant, despite the README.

I guess an alternative would be to just add this example to the README then.